content/version/1/4/code-of-conduct.it.md

```diff
@@ -0,0 +1,83 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/it"]
++++
+
+# Codice di Comportamento dei Collaboratori
+
+## Il nostro impegno
+
+Nell'interesse di promuovere un ambiente aperto ed accogliente, noi come
+collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro 
+progetto e alla nostra comunità un'esperienza libera da molestie per tutti, 
+indipendentemente da età, corporatura, disabilità, etnia, identità ed
+ espressione di genere, livello di esperienza, nazionalità, aspetto fisico, 
+razza, religione, identità od orientamento sessuale.
+
+## Linee guida
+
+Esempi di comportamenti che contribuiscono a creare un ambiente positivo:
+
+* Usare un linguaggio accogliente e inclusivo
+* Essere rispettosi dei diversi punti di vista ed esperienze
+* Accettare con garbo le critiche costruttive
+* Concentrarsi su ciò che è meglio per la comunità
+* Mostrare empatia verso gli altri membri della comunità
+
+Esempi di comportamento inaccettabile:
+
+* Uso di un linguaggio o immagini sessuali ed  attenzioni o avances indesiderate
+* Comportamento da troll o commenti offensivi/dispregiativi e attacchi 
+personali o di ordine politico
+* Molestie pubbliche o private
+* Pubblicazione di informazioni private di un individuo, quali l'indirizzo 
+reale e/o elettronico, senza l'esplicito consenso
```

Comment:
indirizzo fisico e/o elettronico

Reply:
Here I agree with you, better to use "indirizzo fisico e/o elettronico".